cachito/web/api_v1.py

```diff
@@ -226,9 +226,10 @@ def create_request():
 
     package_configs = payload.get("packages", {})
     if "gomod" in pkg_manager_names:
+        go_package_configs = package_configs.get("gomod", {})
```

Comment:
I am wondering why this defaults to  a `dict` and not a `list`. I see you were following the pattern of the other calls, but the `fetch_PKGMANAGER_source` tasks are all relying on lists.

Reply:
https://github.com/release-engineering/cachito/pull/309

Reply:
I assume because that is what @mprahl did initially for some reason. When going back over it, I realized the issue and actually defaulted to `None`, but I will use `[]` as you have in your other PR.